spec/factories/application_qualification.rb

```diff
@@ -116,6 +116,10 @@
         grade { Adviser::ApplicationFormValidations::APPLICABLE_DOMESTIC_DEGREE_GRADES.sample }
         qualification_level { Adviser::ApplicationFormValidations::APPLICABLE_DOMESTIC_DEGREE_LEVELS.sample }
       end
+
+      trait :bachelor do
+        qualification_type { Hesa::DegreeType.all.map(&:name).grep(/Bachelor/).first }
```

Comment:
`Hesa::DegreeType.where(level: :bachelor).first.name?

Reply:
Good suggestion! Done

Reply:
Funny fact: I created the where method 😂